x-pack/metricbeat/module/awsfargate/cloudformation.yml

```diff
@@ -0,0 +1,82 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  SubnetID:
+    Type: String
+Resources:
+  Cluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: test-cloudformation-fargate
+      ClusterSettings:
+        - Name: containerInsights
+          Value: enabled
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: test-cloudformation-fargate-log-group
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: ecsFargateTaskExecutionRole
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: ecs-tasks.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
+      Policies:
+        - PolicyName: !Sub 'EcsTaskExecutionRole-${AWS::StackName}'
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - secretsmanager:GetSecretValue
+                Resource:
+                  - arn:aws:secretsmanager:us-east-2:428152502467:secret:ELASTIC_CLOUD_ID-CcbUG2
+                  - arn:aws:secretsmanager:us-east-2:428152502467:secret:ELASTIC_CLOUD_AUTH-vMz67n
+  TaskDefinition:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      Family: deployment-task-metricbeat
+      Cpu: 256
```

Comment:
Hmm, cpu=256? Does this means 256 vCPUS 🤔 ?

Reply:
Good question :) I searched the AWS [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html) and here is the definition: 
256 meaning `0.25 vCPU`
`Cpu` field represents the number of cpu units used by the task. If you are using the Fargate launch type, this field is required and you must use one of the following values, which determines your range of valid values for the memory parameter:

| Valid Values  | Equivalent vCPU value | Memory Parameters |
| ------------- | ------------- | ------------- |
| 256  | .25 vCPU  | Available memory values: 512 (0.5 GB), 1024 (1 GB), 2048 (2 GB) |
| 512  | .5 vCPU  | Available memory values: 1024 (1 GB), 2048 (2 GB), 3072 (3 GB), 4096 (4 GB) |
| 1024  | 1 vCPU  | Available memory values: 2048 (2 GB), 3072 (3 GB), 4096 (4 GB), 5120 (5 GB), 6144 (6 GB), 7168 (7 GB), 8192 (8 GB) |
| 2048  | 2 vCPU  | Available memory values: Between 4096 (4 GB) and 16384 (16 GB) in increments of 1024 (1 GB) |
| 4096  | 4 vCPU  | Available memory values: Between 8192 (8 GB) and 30720 (30 GB) in increments of 1024 (1 GB) |

Reply:
I would say is fine! At k8s we set `0.1` https://github.com/elastic/beats/blob/0d5ef7b3ccdbd4c83d2a2df018b654164383ecee/deploy/kubernetes/metricbeat-kubernetes.yaml#L180